### ISSUE TITLE
Use of DateTimeFormatter in Logger has raised minimum SDK support to API 26

### ISSUE THREAD

**Issue by darrentaft** [AzureAD/microsoft-authentication-library-common-for-android#1867]
Bug caused by commit:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/commit/ea6e819bcbcdc0357c6cd42d6e210dde17cbe38b

This is now using APIs (including DateTimeFormatter) only available in Android API 26 and greater, causing use on older devices to crash:

```
Caused by java.lang.ClassNotFoundException: Didn't find class "java.time.format.DateTimeFormatter" on path: DexPathList[[zip file "/data/app/app/base.apk", zip file "/data/app/com.foo.bar-1/split_config.arm64_v8a.apk", zip file "/data/app/com.foo.bar-1/split_config.xhdpi.apk"],nativeLibraryDirectories=[/data/app/com.foo.bar-1/lib/arm64, /data/app/com.foo.bar-1/base.apk!/lib/arm64-v8a, /data/app/com.foo.bar-1/split_config.arm64_v8a.apk!/lib/arm64-v8a, /data/app/com.foo.bar-1/split_config.xhdpi.apk!/lib/arm64-v8a, /system/lib64, /vendor/lib64]]
       at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
       at java.lang.ClassLoader.loadClass(ClassLoader.java:380)
       at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
       at com.microsoft.identity.common.java.logging.Logger.<clinit>(Logger.java:72)
       at com.microsoft.identity.common.java.logging.Logger.setLogger(Logger.java)
       at com.microsoft.identity.common.logging.Logger.setAndroidLogger(Logger.java:104)
       at com.microsoft.identity.common.logging.Logger.<clinit>(Logger.java:92)
       at com.microsoft.identity.common.internal.logging.Logger.setAllowLogcat(Logger.java:76)
       at com.microsoft.identity.client.Logger.setEnableLogcatLog(Logger.java:184)
       at com.foo.bar.login.MSALManager.setupLogging(MSALManager.kt:720)
       at com.foo.bar.login.MSALManager.initialiseVars(MSALManager.kt:394)
       at com.foo.bar.login.MSALManager.<init>(MSALManager.kt:75)
       at com.foo.bar.login.MSALManager.<init>(MSALManager.kt)
       at com.foo.bar.login.MSALManager$Companion.init(MSALManager.kt:789)
       at com.foo.bar.login.MSALManager.init(MSALManager.kt)
       at com.foo.bar.async.InitialisationTask.doInBackground(InitialisationTask.java:121)
       at com.foo.bar.async.InitialisationTask.doInBackground(InitialisationTask.java:40)
       at android.os.AsyncTask$2.call(AsyncTask.java:304)
       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
       at java.lang.Thread.run(Thread.java:762)
```

**Comment by shahzaibj** [comment]
@iamgusain FYI

**Comment by iamgusain** [comment]
This is fixed in [8.0.2](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/releases/tag/v8.0.2)
